pkg/frontend/admin_openshiftcluster_kubernetespods_logs.go

```diff
@@ -0,0 +1,92 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+	"github.com/Azure/ARO-RP/pkg/util/restconfig"
+)
+
+func (f *frontend) getAdminKubernetesPodLogs(w http.ResponseWriter, r *http.Request) {
```

Comment:
@anshulvermapatel would you please cover this with unit tests?
We are missing a lot of unit tests in this folder and you would do a great deed to show an example.
If you need any help with the tests, please do not hesisate to reach out and I can help.

Reply:
added a test for this but it is failing .. seems like I did some mistake ..